### PR TITLE
Fixed the typos or mistake in class name in the documentation .

### PR DESCRIPTION
**Pull request by nateshmbhat**
The documentation in the ReadMe says the service name for the class is "MovieService" but the library has the class name as "MoviesService" , you have missed an "s" in the classname in the documentation .

### CODE REVIEW

**Comment by UweTrottmann** [comment]
Thanks for spotting this! Could you please
- remove the merge commit
- not remove the new line at the end of README.md (which adds a pointless change).

File: README.md [hunk 1]
```diff
@@ -54,4 +54,4 @@ Except where noted otherwise, released into the [public domain](UNLICENSE).
 Do not just copy, make it better.
 
 
- [1]: https://square.github.io/retrofit/
\ No newline at end of file
+ [1]: https://square.github.io/retrofit/
```
Comment (UweTrottmann) [inline README.md]: Please undo this change. All files should have an empty line at the end to avoid pointless Git changes.

Comment (nateshmbhat) [reply README.md]: I undid the readme change as u suggested. Please merge PR .

**Comment by UweTrottmann** [comment]
Sorry about the new line at the end of the file. This was actually fine.

I removed the merge commit for you. See #61.

Thanks!
